presentations/en/GCcode-version-control-system-esdc-gc.md

```diff
@@ -0,0 +1,175 @@
+# GCcode
+
+## Version Control for ESDC and GC
+
+### IT Strategy - IITB
+
+2019-xx-xx - CIOEC
+
+---
+
+## The Ask
+
+Provide support in evolving the GCcode version control system (VCS) and project management platform to become officially supported to enable sharing within the GC.
+
+---
+
+## ESDC Context
+
+[ESDC Open Source Software Management Framework](http://dialogue/grp/TAWS-ATST/Collaboration+Architecture+Services/Projects/Artificial+Intelligence+and+Open+Source/OSS/ESDC+Open+Source+Management+Framework+v+1.2.docx)
+
+Endorsed at ARC – September 2019
+
+- Use open source software (OSS)
+- Contribute to OSS communities
+- Publish code as OSS
+
+---
+
+## GC Context
+
+[Directive on Service and Digital](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32601)
+
+[Mandatory Procedures for EA Assessment](https://www.tbs-sct.gc.ca/pol/doc-eng.aspx?id=32602)
+
+- All source code must be released under an appropriate open source software license
+- Share code publicly when appropriate, and when not, share within the GC
+
+--
+
+### GC Digital Standards
+
+> Work in the open by default
+
+Make all non-sensitive data, information, and new code developed in delivery of services open to the outside world for sharing and reuse under an open licence
+
+---
+
+## Current State
+
+Each department manages its own internal VCS/Project management platforms
+
+> ESDC has TFS, SVN and GitLab (Azure)
+
+For open source code and working in the open, SaaS are recomended
+
+> ESDC is using mostly Github.com
+
+--
+
+### GCcode
+
+SSC maintains an unofficial "best effort" internal to the GC instance called GCcode
+
+- ≈3600 projects and ≈3000 users
+- 350+ users from ESDC
+- 50 GC organizations
+- 17k+ issues and 8k merge requests
+
+(May 2019)
+
+---
+
+## Proposal
+
+To increase sharing/reuse in the GC
+
+- Evolve GCcode as an officially supported internal VCS and project management platform for the GC community, with assistance from ESDC (e.g. funding)
+- Make it the default working environment for all source code projects in ESDC
+  - Internal to GC or private projets
+
+Not just a copy of the code, it's where teams work.
+
+A stepping stone to working in the open by default.
+
+---
+
+## More than just version control
+
+- Projects (groups, repositories)
+- VCS (source code, files)
+- Issues (bugs, features, tasks)
+- Kanban (to prioritize/visualize tasks)
+- Continuous Integration (automated testing)
+- Wiki
+- Websites
+- Integration with other systems
+- ..
+
+See [Gitlab Docs](https://docs.gitlab.com/ee/README.html)
+
+---
+
+## Why not SaaS
```

Comment:
The Pros and Cons seem to be inverted :thinking:

Reply:
Agreed, it wasn't clear. I changed to "Advantages of public SaaS" and vice-versa.